sharding-ui/sharding-ui-backend/src/main/java/org/apache/shardingsphere/ui/util/ConfigurationYamlConverter.java

```diff
@@ -38,6 +38,7 @@
 import org.apache.shardingsphere.core.yaml.swapper.impl.ShardingRuleConfigurationYamlSwapper;
 import org.apache.shardingsphere.orchestration.yaml.config.YamlDataSourceConfiguration;
 import org.apache.shardingsphere.orchestration.yaml.swapper.DataSourceConfigurationYamlSwapper;
+import org.springframework.util.StringUtils;
```

Comment:
Use guava's Strings is better than spring's StringUtils

Reply:
Sorry, it's a mistake... this is a map not a string . :<(